<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>53</y>
      <width>521</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_title">
       <property name="text">
        <string>Select Statistics to Display:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_statistics">
       <item>
        <property name="text">
         <string>Lahore Statistics</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Peshawar Statistics</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Combined Statistics</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_generate">
       <property name="text">
        <string>Generate Statistics</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="text_display"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
